styles/styles.css

```
@@ -956,8 +968,7 @@ main > .section > .grid-container {
 }
 
 .theme--hero-indigo {
-  --page-bg-overscroll-color: var(--gradient-indigo-dark-start);
-  --page-bg-main-gradient: var(--gradient-indigo-dark);
+  --page-gradient-start-color: var(--gradient-hero-indigo-start);
   --diamond-fill: rgb(116 91 252);
```

```suggestion
  --page-background-shape-diamond-fill: rgb(116 91 252);
```

or maybe whatever sounds good to you